src/api.js

```diff
@@ -1,8 +1,9 @@
 import axios from 'axios'
+const port = process.env.PORT || process.env.VUE_APP_DEFAULT_PORT
```

Comment:
It seems vue-cli is ignoring `$PORT` and just passing `$VUE_APP_DEFAULT_PORT` :thinking: 
- If you find a way to make it work, we won't need `VUE_APP_DEFAULT_PORT`
- it not, the suggestion below is good enough

```suggestion
const port = process.env.VUE_APP_DEFAULT_PORT || 8001
```